### ISSUE TITLE
Standard input pipeline logs errors

### ISSUE THREAD

**Issue by pronobis** [tensorflow/tensorflow#3949]
The following code:

```
    init_op = tf.group(tf.initialize_all_variables(),
                       tf.initialize_local_variables())
    sess = tf.Session()
    sess.run(init_op)
    coord = tf.train.Coordinator()
    threads = tf.train.start_queue_runners(sess=sess, coord=coord)
    try:
        HERE COMES THE OP READING FROM THE INPUT PIPELINE
    except tf.errors.OutOfRangeError:
        print('Epoch limit reached')
    finally:
        coord.request_stop()
    coord.join(threads)
    sess.close()
```

Finishes without an exception, but generates two log messages marked as errors. 

```
E tensorflow/core/client/tensor_c_api.cc:485] Reached limit of 2
     [[Node: input_producer/limit_epochs/CountUpTo = CountUpTo[T=DT_INT64, _class=["loc:@input_producer/limit_epochs/epochs"], limit=2, _device="/job:localhost/replica:0/task:0/cpu:0"](input_producer/limit_epochs/epochs)]]
E tensorflow/core/client/tensor_c_api.cc:485] FIFOQueue '_0_input_producer' is closed and has insufficient elements (requested 1, current size 0)
     [[Node: ReaderRead = ReaderRead[_class=["loc:@TextLineReader", "loc:@input_producer"], _device="/job:localhost/replica:0/task:0/cpu:0"](TextLineReader, input_producer)]]
Epoch limit reached
```

Shouldn't these be warnings or simply hidden?
### Environment info

Operating System: Ubuntu 16.04

Installed version of CUDA and cuDNN: 
8.0.27 and 5.0.5

Installed from source: 
Commit hash: 3cb39956e622b322e43547cf2b6e337020643f21 (v0.10.0rc0)

Bazel:
Build label: 0.3.0
Build target: bazel-out/local-fastbuild/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
Build time: Fri Jun 10 11:38:23 2016 (1465558703)
Build timestamp: 1465558703
Build timestamp as int: 1465558703

**Comment by prb12** [comment]
@davidsoergel Looks like input handling ops return an error status on EOF which causes session.run to abort graph execution and log an error?

I don't see that this would be easy to change?

**Comment by drpngx** [comment]
Is this something that we still want to do?

**Comment by drpngx** [comment]
Closing due to inactivity. Feel free to re-open if you would like us to look again.
